WordPress/src/main/java/org/wordpress/android/WordPress.java

```diff
@@ -383,7 +383,7 @@ public static void registerForCloudMessaging(Context context) {
      * select the first one.
      */
     public static Blog getCurrentBlog() {
-        if (currentBlog == null || !wpDB.isDotComBlogVisible(currentBlog.getRemoteBlogId())) {
+        if (currentBlog == null) {
```

Comment:
This is obviously a very "impactful" change. However, I decided remove it because we can now select hidden sites. From what I saw, there is nothing catastrophic for calling a hidden blog anywhere. If that's not the case, I'll change it back.